regen/HeaderParser.pm

```diff
@@ -794,6 +794,7 @@ sub tidy_embed_fnc_entry {
         return_type => $ret,
         name        => $name,
         args        => \@args,
+        start_line_num => $line_data->{start_line_num},
```

Comment:
The preceding lines should be reïndented to make the `=>` all line up.

Reply:
Changed as requested